recipes/freeglut/all/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_definitions("-DFREEGLUT_LIB_PRAGMAS=0")
```

Comment:
You added this define in package_info already when os=Windows.
Is this line needed here?

Reply:
I think so, package_info is used when consuming freeglut, but this CMakeLists is used when building freeglut

Reply:
Oh yes indeed.
Sorry, I thought this was test_package's `CMakeLists.txt`